esmvalcore/cmor/_fixes/obs4mips/airs_2_1.py

```diff
@@ -0,0 +1,35 @@
+
+"""Fixes for obs4mips dataset AIRS-2-1."""
+import iris
+from iris.cube import CubeList
+from cf_units import Unit
+
+from ..fix import Fix
+
+
+class AllVars(Fix):
+    """Common fixes to all vars."""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata.
+
+        Change unit of coordinate plev from hPa to Pa
+
+        Parameters
+        ----------
+        cube: iris.cube.CubeList
+
+        Returns
+        -------
+        iris.cube.Cube
+
+        """
+        for cube in cubes:
+            try:
+                plev = cube.coord('air_pressure')
+            except iris.exceptions.CoordinateNotFoundError:
+                continue
+            else:
+                plev.units = Unit('Pa')
```

Comment:
It would be good to add some check here that the units are indeed wrong. Maybe something like

```py
if plev.points[0] > 10000.0:
    plev.units = 'Pa'
```

Assuming the first entry in the array is the surface level and the values are in "hPa" but the units says "Pa".